### PR TITLE
HOLD FOR RELEASE: Update About Customers docs page with new instance rows info

### PR DESCRIPTION
**Pull request by alicenstar**
Related story: https://app.shortcut.com/replicated/story/65023/customer-reporting-page-instances-list

We have updated the way the instance rows on the Customer Reporting page look. They now display additional info, as well as a mini uptime graph and the uptime ratio for the instance. This PR adds new screenshots and attempts to accurately update the docs related to this page.

Preview: https://deploy-preview-902--replicated-docs.netlify.app/vendor/licenses-about

### CODE REVIEW

**Review by paigecalvert — APPROVED** [review]
@alicenstar thank you!! Looks great. I ended up moving everything into a new "About the Customer Reporting Page" subsection just because the list was feeling long. Also tried to group the list a bit into common categories of items to make it more skim-able.

Approving so that this is ready to go as needed, though feel free to make any edits on my changes as needed.

**Review by jonquil2002 — APPROVED** [review]
lgtm

File: docs/vendor/licenses-about.md [hunk 1]
```diff
@@ -72,25 +74,60 @@ This is a convenience feature for how licenses are displayed in the vendor porta
...
+     [View a larger version of this image](/images/kurl-instance-row.png)
+
+   * The app manager version running in the cluster
+   * The cloud provider and region, if applicable.
+
+* Instance uptime data, including:
+
+   * The timestamp of the last recorded check-in for the instance. A check-in is recorded when any of the following occur:
+   
+      <AppCheckin/>
```
Comment (jonquil2002) [inline docs/vendor/licenses-about.md:128]: This should be indented one more level.

File: docs/vendor/licenses-about.md [hunk 2]
```diff
@@ -72,25 +74,60 @@ This is a convenience feature for how licenses are displayed in the vendor porta
...
 
-  For each active instance, the **Reporting** page displays:
-  * The application version 
-  * The app manager version
-  * The status of the instance
-  * The timestamps of the earliest and latest events recorded for the instance
-  * The Kubernetes distribution for the cluster where the instance is installed
-  * A **View details** button that opens the **Instance details** page. The **Instance details** page displays additional event data and computed metrics to help you understand the performance and status of each active application instance.
+   * An uptime graph of the previous two weeks. For more information about how the vendor portal determines uptime, see [Instance Uptime](https://docs.replicated.com/vendor/instance-insights-details#instance-uptime) in _Viewing Instance Details_.
```
Comment (jonquil2002) [inline docs/vendor/licenses-about.md:130]: This should also be indented one more level.

File: docs/vendor/licenses-about.md [hunk 3]
```diff
@@ -72,25 +74,60 @@ This is a convenience feature for how licenses are displayed in the vendor porta
...
+
+The **Reporting** page displays data about the active application instances associated with each customer. The following shows an example of the **Reporting** page for a customer:
+
+![Customer reporting page showing one active instance](/images/customer-reporting-page.png)
+
+[View a larger version of this image](/images/customer-reporting-page.png)
+
+You can click any of the rows on the **Reporting** page to open the **Instance details** page. The **Instance details** page displays additional event data and computed metrics to help you understand the performance and status of each active application instance. For more information, see [Viewing Instance Details](https://docs.replicated.com/vendor/instance-insights-details).
+
+The **Reporting** page displays the following details about each active instance:
```
Comment (jonquil2002) [inline docs/vendor/licenses-about.md:100]: Minor note - this list sometimes uses periods and sometimes doesn't. Suggest making it consistent one way or the other.

**Pull request merged**